about.html

```diff
@@ -0,0 +1,194 @@
+            feature 24 pairs, including 20 pairs automatically entered based on ITTF World Table Tennis Ranking in the world. <br>The remaining spots are filled by four (4) host wildcards.
+          </p>
+          <p class="description-about-mobile">
+            Player entry is mandatory by world ranking using the ITTF Table Tennis World Rankings released four weeks prior to the start of the event.
+            The singles entry list will be announced on 8 February 2023 with the doubles entries lists to be announced one week later.
+          </p>
+          <p class="description-about-desktop">
+            
+          </p>
+        </div>    
+        
+        <p class="contact">Please contact us per Email for any further questions about Singapore Smash 2023</p>
+        <p class="email">event@smashwtt.com</p>
+      </div>
+    </section>
+
+    <section class="about-logo">
+      <div class="logo-title">
+        <h3 class="logo-title-h3">SINGAPORE SMASH 2023</h3>
+        <h3 class="logo-title-h3">WTT 2023 Logo</h3>
+        <h3 class="logo-title-h3-desktop">WTT SINGAPORE SMASH 2023 Logo</h3>
+      </div>
+      <div class="about-bar"></div>
+      <div class="about-ministries">
+        <p class="about-logo-p">
+          Will Defending Champions Chen Meng and Fan Zhendong claim the crown again? Or will one of the best 128 Men’s and Women’s Singles players set to descend on
+           Singapore steal their thunder as they compete for their share of USD 2,000,000 in prize money and 2000 World Ranking points.
+        </p>
+        <p class="about-logo-p-desktop">
+          Will Defending Champions Chen Meng and Fan Zhendong claim the crown again? Or will one of the best 128 Men’s and Women’s Singles players set to descend on
+           Singapore steal their thunder as they compete for their share of USD 2,000,000 in prize money and 2000 World Ranking points. <br>
+          The logo of the Global Summit 2023 was designed and chose according to the preferences of our sponsors as following:
+        </p>
+      </div>
+      <div class="about-img">
+        <img class="logo-img" src="Images/wtt_orange_inverse_new.png" alt="wtt logo">
+     </div>
+    </section>
+
+    <section class="see-past">
+      <h5 class="see-title">
+        See the past WTT EVENTS
+      </h5>
+      <div class="about-bar"></div>
+      <p class="see-paragrah">
+        Take a look at the last two WTT EVENTS  
+        which took place in Dubai and in Lisboa.
+      </p>
+      <div class="conference-images">
+        <div class="conference1">
+          <h6 class="conference-year">2022</h6>
+          <p class="conference-prev">SMASH WTT in Dubai</p>
+        </div>
+        <div class="conference2">
+          <h6 class="conference-year">2021</h6>
+          <p class="conference-prev">SMASH WTT  in Lisboa</p>
+        </div>
+        
```

Comment:
- [ ] From the project requirements, you should stick to [Cindy's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), But as you can see, the blurred images, logo are not showing which makes your project design not matching [Cindy's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). So please make sure that you adjust the style of your design to match **_exactly_** like Cindy's design :+1: 

### Your Design
![image](https://user-images.githubusercontent.com/92303891/227748837-676fe509-7118-4e84-b1ad-0f251b6fec97.png)


### Cindy's design
![image](https://user-images.githubusercontent.com/92303891/227748815-19cf08a6-b8c7-44f5-bab5-2b7031841bb7.png)